kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java

```diff
@@ -17,6 +17,7 @@
 	public static final int BAD_CONFIGURATION = 4002;
 	public static final int INTERNAL_INCONSISTENCY = 4003;
 	public static final int ORDER_NOT_FOUND = 4004;
+	public static final int ACCOUNT_NOT_LOGGED_IN = 4005;
```

Comment:
Just for knowledge - why you choose 4000+ or it is an arbitrary decision

Reply:
I didn't